tutorials-v5/heom/heom-1d-spin-bath-model-ohmic-fitting.md

```diff
@@ -76,61 +73,7 @@ mp.pretty = True
 
 ## Helper functions
 
-Let's define some helper functions for plotting results and timing how long operations take:
-
-```{code-cell} ipython3
-def coth(x):
-    """ Vectorized hyperbolic cotangent of x. """
-    return 1. / np.tanh(x)
-```
-
-```{code-cell} ipython3
-def plot_result_expectations(plots, axes=None):
-    """ Plot the expectation values of operators as functions of time.
-
-        Each plot in plots consists of (solver_result,
-        measurement_operation, color, label).
-    """
-    if axes is None:
-        fig, axes = plt.subplots(1, 1, sharex=True, figsize=(8, 8))
-        fig_created = True
-    else:
-        fig = None
-        fig_created = False
-
-    # add kw arguments to each plot if missing
-    plots = [p if len(p) == 5 else p + ({},) for p in plots]
-    for result, m_op, color, label, kw in plots:
-        exp = np.real(expect(result.states, m_op))
-        kw.setdefault("linewidth", 2)
-        if color == 'rand':
-            axes.plot(
-                result.times, exp,
-                c=np.random.rand(3,), label=label, **kw,
-            )
-        else:
-            axes.plot(result.times, exp, color, label=label, **kw)
-
-    if fig_created:
-        axes.legend(loc=0, fontsize=12)
-        axes.set_xlabel("t", fontsize=28)
-
-    return fig
-```
-
-```{code-cell} ipython3
-@contextlib.contextmanager
-def timer(label):
-    """ Simple utility for timing functions:
-
-        with timer("name"):
-            ... code to time ...
-    """
-    start = time.time()
-    yield
-    end = time.time()
-    print(f"{label}: {end - start}")
-```
+Let's define some helper functions for plotting the resutls
```

Comment:
Suggestion: delete

Reply:
Github is formatting this weirdly, I meant to delete lines 73-76 since there are no helper functions anymore